src/test/resources/DatabaseQueueComponentTest-context.xml

```diff
@@ -39,9 +39,13 @@
       <to id="producer_mock_to" uri="mock:mock_endpoint"/>
     </route>
     <route id="consumer_test_route">
-      <from id="consumer_from" uri="sdpqueue:message_queue?dataSource=sdpqDataSource"/>
+      <from id="consumer_from" uri="sdpqueue:message_queue?dataSource=sdpqDataSource&amp;initialDelay=7"/>
       <to id="consumer_mock_to" uri="mock:mock_endpoint"/>
     </route>
+    <route id="delayed_consumer_test_route">
```

Comment:
I will add a third well-named test route, but to clarify: by default (in a scheduledbatchpollingconsumer) the initialDelay is 1 second and delay is a half second; initialDelay must be set to 0 in order to test the delay parameter in the second test route.